### PR TITLE
Update alter-database-transact-sql.md

### PR DESCRIPTION
**Pull request by charithcaldera**
ALTER DATABASE MODIFY NAME does not work with Azure Synapse Analytics Dedicated SQL Pool
[Test Scenario - Rename SQL Pools.docx](https://github.com/MicrosoftDocs/sql-docs/files/8817792/Test.Scenario.-.Rename.SQL.Pools.docx)
s

### CODE REVIEW

**Comment by PRMerger-2** [comment]
@charithcaldera : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by charithcaldera** [comment]
Hi @WilliamDAssafMSFT , 
ALTER DATABASE MODIFY NAME works for Legacy standalone Dedicated SQL Pools (formerly SQL DW), but it does not work for Dedicated SQL Pools in Azure Synapse workspace.
Document attached.
[Test Scenario - Rename SQL Pools.docx](https://github.com/MicrosoftDocs/sql-docs/files/8817897/Test.Scenario.-.Rename.SQL.Pools.docx)

**Comment by jborsecnik** [comment]
@WilliamDAssafMSFT, if you approve the changes, please add a `#sign-off` comment to signal that the PR is ready for the review team to merge. Thanks.

#label:"aq-pr-triaged"

**Comment by WilliamDAssafMSFT** [comment]
> Hi @WilliamDAssafMSFT , ALTER DATABASE MODIFY NAME works for Legacy standalone Dedicated SQL Pools (formerly SQL DW), but it does not work for Dedicated SQL Pools in Azure Synapse workspace. Document attached. [Test Scenario - Rename SQL Pools.docx](https://github.com/MicrosoftDocs/sql-docs/files/8817897/Test.Scenario.-.Rename.SQL.Pools.docx)

Based on that information the article is accurate. The comments on line 1044 are specific as to what scenarios `ALTER DATABASE MODIFY NAME` is supported. These changes as proposed would be too broad, as `ALTER DATABASE MODIFY NAME` is supported in some dedicated SQL pools.

**Comment by charithcaldera** [comment]
@WilliamDAssafMSFT , 
This appears to be making a confusion here, 

Azure Synapse SQL Dedicated Pool and Standalone Dedicated Pools (Formerly SQL DW) are  treated as two. 

In this public document, in line 966, its mentioned as follows which can make a confusion - 
  "In Azure Synapse, `ALTER DATABASE` modifies the name, maximum size, or service objective for a database." 

Since Azure Synapse Analytics Dedicated SQL Pools does not supported on "ALTER DATABASE MODIFY NAME" according to line 1046, the two lines (966 and 1046) having a conflict there. 

The problem is not about the Standalone Dedicated Pools (Formerly SQL DW), but the Dedicated Pools in Azure Synapse Workspace. 
I hope its clear.

File: docs/t-sql/statements/alter-database-transact-sql.md [hunk 1]
```diff
@@ -963,7 +963,7 @@ ALTER DATABASE WideWorldImporters
 
 ## Overview: Azure Synapse Analytics
 
-In Azure Synapse, `ALTER DATABASE` modifies the name, maximum size, or service objective for a database.
+In Azure Synapse Dedicated SQL Pools, `ALTER DATABASE` modifies the maximum size, or service objective for a database.
```
Comment (WilliamDAssafMSFT) [inline docs/t-sql/statements/alter-database-transact-sql.md:966]:
```suggestion
In Azure Synapse, `ALTER DATABASE` modifies certain configuration options of a pool.

```

File: docs/t-sql/statements/alter-database-transact-sql.md [hunk 2]
```diff
@@ -976,8 +976,7 @@ Provides the syntax and related information for changing the attributes of a dat
 ```syntaxsql
 ALTER DATABASE { database_name | CURRENT }
 {
-  MODIFY NAME = new_database_name
-| MODIFY ( <edition_option> [, ... n] )
+  MODIFY ( <edition_option> [, ... n] )
```
Comment (WilliamDAssafMSFT) [inline docs/t-sql/statements/alter-database-transact-sql.md:979]:
```suggestion
  MODIFY NAME = new_database_name
| MODIFY ( <edition_option> [, ... n] )
```
Reverting this change as it does apply to dedicated SQL pools formerly SQL DW created via the Azure portal, and we clarify this later in the arguments section.

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]
I understand. I've updated the line near the top of the article to be more general. I've reverted the change on line 966. Please commit these two Suggestions, thanks!

File: docs/t-sql/statements/alter-database-transact-sql.md (hunk 2 above)
Comment (charithcaldera) [reply docs/t-sql/statements/alter-database-transact-sql.md:979]: #sign-off

**Comment by PRMerger20** [comment]
@charithcaldera: I'm sorry - only the author of this article, @WilliamDAssafMSFT, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: docs/t-sql/statements/alter-database-transact-sql.md (hunk 1 above)
Comment (charithcaldera) [reply docs/t-sql/statements/alter-database-transact-sql.md:966]: I agree with this comment.

**Comment by WilliamDAssafMSFT** [comment]
#hold-off 
@charithcaldera you should see options beneath each of my suggestions to "Commit suggestion". Go ahead and commit those changes. You'll know for sure they're in when they are reflected in the Files Changed tab.
image

**Comment by PRMerger14** [comment]
@charithcaldera : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@charithcaldera : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by charithcaldera** [comment]
> 

Hi @WilliamDAssafMSFT ,
Done

**Comment by WilliamDAssafMSFT** [comment]
Thanks @charithcaldera!! 
#sign-off

**Review by WilliamDAssafMSFT — APPROVED** [review]

**Comment by jborsecnik** [comment]
#assign:@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger10** [comment]
The following users don't have valid GitHub IDs, or are not collaborators on this repo: microsoftdocs/public-repo-pr-review-team

**Pull request merged**